app/scripts/locales/fr-FR.json

```diff
@@ -426,6 +426,8 @@
   "setFileBackupNowWorking": "En cours...",
   "setFileBackupError": "Erreur sauvegarde",
   "setFileBackupErrorDescription": "Erreur à l'écriture du fichier",
+  "setFileBackupErrorIsDir": "Chemin de sauvegarde invalide",
```

Comment:
Please don't edit these files: they come from OneSky and will be overwritten in any case.

Reply:
Oh, okay. Didn't know that. So just add something to the base.json and the rest will come from OneSky?

Reply:
Yes, correct.